ci/deps/azure-37-minimum_versions.yaml

```diff
@@ -15,16 +15,16 @@ dependencies:
   # pandas dependencies
   - beautifulsoup4=4.6.0
   - bottleneck=1.2.1
-  - jinja2=2.8
+  - jinja2=2.10
   - numba=0.46.0
-  - numexpr=2.6.2
-  - numpy=1.15.4
+  - numexpr=2.6.8
+  - numpy=1.16.0
   - openpyxl=2.5.7
-  - pytables=3.4.3
-  - python-dateutil=2.7.3
-  - pytz=2017.2
+  - pytables=3.5.1
+  - python-dateutil=2.7.4
+  - pytz=2018.3
   - scipy=1.2
   - xlrd=1.1.0
-  - xlsxwriter=0.9.8
-  - xlwt=1.2.0
+  - xlsxwriter=1.0.2
+  - xlwt=1.3.0
   - html5lib=1.0.1
```

Comment:
Are all those version increases in this file needed? (as this should test our minimal supported versions, generally, so if we need to update some, we might (depending on the reason) also need to update our listedd minimum versions)

Reply:
> Are all those version increases in this file needed? (as this should test our minimal supported versions, generally, so if we need to update some, we might (depending on the reason) also need to update our listedd minimum versions)

I double-checked again and lowered the versions for some of them in the new commit. Now they are the minimum versions without conflicts.